tests/tests/wgpu-validation/util.rs

```diff
@@ -1,6 +1,7 @@
 //! Tests of [`wgpu::util`].
 
 use nanorand::Rng;
+use wgpu::BufferUsages;
```

Comment:
I don't think we have a set style on this yet, but since the file currently uses all qualified names, please stick to that; that is, remove this `use`.